net-libs/libomemo-c/libomemo-c-0.5.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="XMPP fork of Signal Protocol C Library supporting XEP-0384 OMEMO"
+HOMEPAGE="https://github.com/dino/libomemo-c/"
+SRC_URI="https://github.com/dino/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
Could you look closer into this? For example, 
https://github.com/dino/libomemo-c/blob/omemo/src/vpool.c looks more like MIT or BSD (I never remember which one).

There may be other files with different licenses.

Reply:
Indeed, found 3 more licenses. Fixed.